packages/block-library/src/media-text/media-container.js

```diff
@@ -62,22 +62,18 @@ function ToolbarEditButton( {
 	onSelectMedia,
 	toggleUseFeaturedImage,
 	useFeaturedImage,
-	featuredImageURL,
 } ) {
 	return (
 		<BlockControls group="other">
 			<MediaReplaceFlow
 				mediaId={ mediaId }
-				mediaUrl={
-					useFeaturedImage && featuredImageURL
-						? featuredImageURL
-						: mediaUrl
-				}
```

Comment:
This looks unrelated. Can you explain why we're removing special handling for featured images?

Reply:
Makes sense!